core/src/main/scala/org/scalarelational/WrappedString.scala

```diff
@@ -8,6 +8,6 @@ package org.scalarelational
  *
  * @author Matt Hicks <matt@outr.com>
  */
-trait WrappedString {
+/*trait WrappedString {
```

Comment:
You can go ahead and delete WrappedString. It uses Reflection anyway and should be removed.